earth_enterprise/src/fusion/rasterfuse/TmeshGenerator.cpp

```diff
@@ -56,7 +56,10 @@ TmeshGenerator::Generate(float *srcSamples,
                          const khOffset<uint32> &wantOffset,
                          const khTileAddr &tmeshAddr,
                          etArray<uchar> &compressed,
-                         const size_t reserve_size)
+                         const size_t reserve_size,
+                         bool decimate,
+                         float decimation_threshold
```

Comment:
why not "double" ?  everything else in the error calculation is double

Reply:
But a double takes up twice as much memory .. I think I forgot I was writing server code again